examples/common/hdr_reader.cpp

```diff
@@ -81,7 +81,7 @@ unsigned char *loadHdr(const char *filename, HdrInfo *info, bool convertToFloat)
     while(true) {
         if (! fgets(buffer, MAXLINE, fp)) goto error;
         if (buffer[0] == '\n') break;
-        if (buffer[0] == '\r' && buffer[0] == '\n') break;
+        if (buffer[0] == '\r' || buffer[0] == '\n') break;
```

Comment:
With the change to "||" the second condition is not necessary as it always evaluates to false given line 83.  Either the new line 84 makes line 83 redundant, in which case 83 can be removed, or line 84 can be reduced to its first condition.